### PR TITLE
Unit tests

### LINKED ISSUE #3: Unit Tests of utility functions
**Description**
The utility functions don't have tests. Add the Jest dependency and add additional configurations to be able to create `.spec.ts` files.

Simply create the tests for each of the utility functions and which ever edge cases you can find.

Also add Script for executing test suite.

**Links/References**
https://jestjs.io/docs/getting-started

**Tasks**

- [ ] Add dependency
- [ ] Configure Jest for project
- [ ] Add Unit tests for utility funcs
- [ ] Add Script command to `package.json`

**Additional Context**

### PR DESCRIPTION
**Pull request by TamaraBogantseva**
**Checklist**

- [x] Resolves  #3 
- [x] The [code of conduct](../blob/master/.github/CODE_OF_CONDUCT.md) is respected
- [x] Labelled and assigned a reviewer

**Reviewer**

- [ ] I've checked out the code and tested it myself.

**Changes**

- configure jest for unit tests.
- create link test
- shuffle participant test
- change create link test and function = pass encoding func as a param
- validate shuffle test
- form serializer test

### CODE REVIEW

File: src/lib/util/create-link.ts [hunk 1]
```diff
@@ -2,10 +2,9 @@ import type { Participant } from '$lib/stores/participants-store';
 
 type linkParams = { participant: Participant; assignment: Participant };
 
-export function createLink(url: string, { participant, assignment }: linkParams): string {
+export function createLink(url: string, { participant, assignment }: linkParams, encode:Function = btoa): string {
```
Comment (cowglow) [inline src/lib/util/create-link.ts]: Yes, now we can pass any kind of encoder we want. Let's change the name to `encoder` please.

File: src/lib/util/shuffle-participants.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const randomizeParticipants = (participants: Participant[]): Participant[
 		const temp = _participants[randomIndex];
 
 		_participants[randomIndex] = _participants[i];
-		_participants[i] = temp;
+		_participants[i] = temp;
```
Comment (cowglow) [inline src/lib/util/shuffle-participants.ts]: Check for the `tab space`

File: src/lib/util/validate-shuffle.spec.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { validateShuffle } from "./validate-shuffle";
+import { demoParticipants } from '../demo-participants'
+
+describe('check that func return what we expected', () => {
```
Comment (cowglow) [inline src/lib/util/validate-shuffle.spec.ts]: Maybe adding the name of the function will help better read the jest log.

**Review by cowglow — CHANGES_REQUESTED** [review]
Great work. Let's try to name the function on the initial `describe` block. 
And always check for dead code like our friend the `tab space` that we found.
